src/tools/pub/custom-package-repositories.md

```diff
@@ -3,16 +3,17 @@ title: Custom package repositories
 description: "How Dart's package management tool, pub, works with custom package repositories."
 ---
 
-The `pub` tool has support for third-party package repositories. A package
+The `dart pub` tool has support for third-party package repositories. A package
 repository is an HTTP server from which the `pub` client can fetch packages.
 The official package repository is [pub.dev]({{site.pub}}), this repository
-is by the `pub` tool by default. A package repository is identified by a
-_hosted-url_ (e.g. `https://pub.example.com/my-account/`). 
-
-Third-party package servers can be useful for hosting private packages. It is
-also common to use [git-dependencies](/tools/pub/dependencies#git-packages) for hosting
-private packages, however, the `pub` does not support resolving versions against
-a git-repository it just fetches a specific revision of the git-repository.
+is by the `dart pub` tool by default. A package repository is identified by a
+_hosted-url_ (e.g. `https://dart-packages.example.com/my-account/`). 
```

Comment:
Writing guidelines, we don't use "e.g" or "i.e." replace with "for example" or "such as" as appropriate.